src/engine/engine.rs

```diff
@@ -16,8 +16,7 @@ use devicemapper::{Bytes, Device, Sectors};
 use crate::{
     engine::types::{
         BlockDevState, BlockDevTier, CreateAction, DeleteAction, DevUuid, FilesystemUuid,
-        FreeSpaceState, MaybeDbusPath, Name, PoolExtendState, PoolState, PoolUuid, RenameAction,
-        SetCreateAction, SetDeleteAction,
+        MaybeDbusPath, Name, PoolUuid, RenameAction, SetCreateAction, SetDeleteAction,
```

Comment:
You should be able to remove ```BlockDevState``` and ```BlockDev::state```, I think.

Reply:
Resolved.